src/vs/workbench/contrib/inlineChat/browser/inlineChatActions.ts

```diff
@@ -450,21 +452,28 @@ export class ToggleInlineDiff extends AbstractInlineChatAction {
 	constructor() {
 		super({
 			id: 'inlineChat.toggleDiff',
-			title: localize('toggleDiff', 'Toggle Diff'),
-			icon: Codicon.diff,
-			precondition: ContextKeyExpr.and(CTX_INLINE_CHAT_VISIBLE, CTX_INLINE_CHAT_DID_EDIT),
-			toggled: { condition: CTX_INLINE_CHAT_SHOWING_DIFF, title: localize('toggleDiff2', "Show Inline Diff") },
-			menu: {
-				id: MENU_INLINE_CHAT_WIDGET_DISCARD,
-				when: CTX_INLINE_CHAT_EDIT_MODE.notEqualsTo(EditMode.Preview),
-				group: '1_config',
-				order: 9
-			}
+			title: {
+				value: localize('toggleDiff', 'Toggle Diff'),
+				mnemonicTitle: localize({ key: 'miToggleDiff', comment: ['&& denotes a mnemonic'] }, "&&Toggle Diff"),
+				original: 'Toggle Diff',
+			},
+			category: Categories.View,
+			toggled: {
+				condition: ContextKeyExpr.equals('config.inlineChat.showDiff', true),
+				title: localize('toggleDiff2', "Toggle Diff"),
```

Comment:
This `title` allows for more specific wording that goes with the checkmark, e.g "Show Diff"